models/Product.js

```diff
@@ -1,53 +1,60 @@
 const mongoose = require('mongoose');
 const Language = require('./Language').schema;
 const PrimaryImage = require('./PrimaryImage').schema;
-const Size = require('./Size').schema;
+const ImageSet = require('../modules/common/ImageSet').schema;
+const Color = require('./Color').schema;
 
 const productSchema = new mongoose.Schema({
-  category: {
+  subcategory: {
     type: mongoose.Schema.Types.ObjectId,
     ref: 'Category',
   },
   name: [Language],
   description: [Language],
+  mainMaterial: [Language],
+  innerMaterial: [Language],
```

Comment:
This one doesn't have an entry in the materials collection, because its name is literary all we know about it and it's the same for all products. Adding a link here would mean sacrificing performance for no gain